.github/Csr.sh

```diff
@@ -0,0 +1,168 @@
+#!/bin/bash
+
+
+# kubi private key 
+openssl ecparam -genkey -name secp521r1 -noout -out /tmp/ecdsa-key.pem
```

Comment:
it would probably be better to build a folder from mktemp

Reply:
done !